.github/workflows/ci.yml

```diff
@@ -301,15 +306,22 @@ jobs:
       contents: read
       id-token: write
     steps:
+      - id: get-secrets
+        uses: grafana/shared-workflows/actions/get-vault-secrets@28361cdb22223e5f1e34358c86c20908e7248760 # get-vault-secrets-v1.1.0
+        with:
+          common_secrets: |
+            GITHUB_APP_ID=plugins-platform-bot-app:app_id
```

Comment:
we should use repo-specific secrets not common_secrets

Reply:
Sorry, should be sorted now.